doofinder/includes/class-migration.php

```diff
@@ -40,6 +44,22 @@ public static function migrate()
         }
     }
 
+    /**
+     * Initialize the migration
+     *
+     * @return void
+     */
+    private static function initialize_migration()
```

Comment:
private and static?

Reply:
It is a static class and the initialize function should be called only from the public function migrate()

Reply:
I don't understand this. You could always call it for itself using $this->initialize_migration(), and the same goes to the private attributes mentioned before

Reply:
It is a very simple and static class.
To do what you suggest, we should add a constructor to instantiate the class, what isn't needed for this tiny class IMO.